### PR TITLE
WW-4845 pass all current tests with java 9

### PR DESCRIPTION
**Pull request by yasserzamani**

### CODE REVIEW

**Comment by yasserzamani** [comment]
@apache/struts-committers , As you see travis jdk9 can build and pass all tests :)

I workarounded coveralls-java9 problem but it seems coveralls doesn't union coverages from three travis builds, openjdk7, oraclejdk8 and oraclejdk9 :( so fails this PR with new missed lines while they're not and they are covered in oraclejdk9's build. Please let me know if you have any idea how to fix this.

**Comment by yasserzamani** [comment]
Hmmm no, travis jdk9 didn't pass any report and have failed with:

>  Failed to execute goal morg.codehaus.mojo:cobertura-maven-plugin:2.7:instrument
Execution default-cli of goal org.codehaus.mojo:cobertura-maven-plugin:2.7:instrument failed: Plugin org.codehaus.mojo:cobertura-maven-plugin:2.7 or one of its dependencies could not be resolved: Could not find artifact com.sun:tools:jar:0 at specified path /usr/lib/jvm/java-9-oracle/../lib/tools.jar

File: core/src/test/java/com/opensymphony/xwork2/ognl/OgnlUtilTest.java [hunk 1]
```diff
@@ -374,8 +375,18 @@ public void testSetPropertiesDate() {
...
+            Unicode Consortium's Common Locale Data Repository (CLDR). The short date-time format is ‹{1}, {0}› in the
+            CLDR locale, as opposed to {1} {0} in the JRE locale data.
+            Please refer : http://www.oracle.com/technetwork/java/javase/9-relnote-issues-3704069.html#JDK-8008577 */
+            props.put("event", "18/10/2006, 14:23:45");
+            props.put("meeting", "09/09/2006, 14:30");
+        }
+        else {
+            props.put("event", "18/10/2006 14:23:45");
+            props.put("meeting", "09/09/2006 14:30");
+        }
```
Comment (lukaszlenart) [inline core/src/test/java/com/opensymphony/xwork2/ognl/OgnlUtilTest.java]: I'm not a fan of such conditional testing but I have no better idea right now. So let's megre this and move forward, thank you :)

**Comment by lukaszlenart** [comment]
@apache/struts-committers I think we are good here, any objections?

**Comment by lukaszlenart** [comment]
LGTM 👍

**Pull request merged**